src/app/components/Dashboard/Dashboard.tsx

```diff
@@ -81,4 +105,14 @@ const Card = styled.li`
   border-radius: 15px;
   box-shadow: rgba(50, 50, 93, 0.25) 0px 2px 5px -1px,
     rgba(0, 0, 0, 0.3) 0px 1px 3px -1px;
+  word-wrap: break-word;
+`
+const FormWrapper = styled.div`
+  z-index: 1000;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.7);
```

Comment:
This wrapper seems to be on top of the form, I can't click inside the form's inputs